license.go

```diff
@@ -35,6 +35,9 @@ func (l License) Validate() error {
 	if l.CreatedAt.UTC().After(now) {
 		return errors.Wrap(ErrMalformedEntity, ErrIssuedAt)
 	}
+	if !l.Active {
```

Comment:
You have the same change in separate PR #25. Pls discard your change in #25 or close this PR.